addons/website_sale/static/src/js/widget_multirange_price_selector.js

```diff
@@ -0,0 +1,28 @@
+odoo.define('website_sale.multirange_price_selector', function (require) {
+'use strict';
+
+var publicWidget = require('web.public.widget');
+
+publicWidget.registry.multirangePriceSelector = publicWidget.Widget.extend({
+    selector: '#o_shop_price_range',
+    events: {
+        'newRangeValue input[type="range"]': '_onPriceRangeSelected',
+    },
+    //----------------------------------------------------------------------
+    // Handlers
+    //----------------------------------------------------------------------
+
+    /**
+     * Apply the new range
```

Comment:
Appl**ies** the new range. + linebreak

-> But in fact, no comment, the name of the function already tells that.